systers_portal/templates/community/edit_profile.html

```diff
@@ -22,4 +22,4 @@ <h1>Edit {{ community }} profile</h1>
 
 {% block community_footer %}
   {% include 'community/snippets/footer.html' %}
-{% endblock %}
+{% endblock %}
```

Comment:
You could remove this change.

Reply:
I tried undoing it but it still reflects in the PR. There is no difference between the statements though.

Reply:
Does it still exist if you add a new line after `{% endblock %}`?